### PR TITLE
feat: modified documentation

### PR DESCRIPTION
**Pull request by antonio-marasescu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,34 +13,59 @@
...
+- You need to have [Git](https://git-scm.com) installed on your computer. Have a look at [this video](https://www.youtube.com/watch?v=HkdAHXoRtos) for a quick crash course.
 
 In the `backend` folder you can find a server-side implementation for the online shop. To run this server perform the following:
 - Open the `backend` folder in your terminal
 - `npm ci` (only needed the first time your run the server)
 - `npm start`
 - Open [http://localhost:3000](http://localhost:3000) in your browser.
 
+
+## Time Bookings
```
Comment (antim-prisacaru) [inline README.md:54]: Perhaps it's not such a great idea to mention bookings in the training, considering that not only the interns will be performing it, but also colleagues that for example want to switch from SAP to actual programming.

File: README.md [hunk 2]
```diff
@@ -12,34 +13,59 @@
...
+
+**In order to request a code review from the trainers**, you must [open a pull request](https://help.github.com/en/articles/creating-a-pull-request) from the `develop` to the `master` branch. Inform them in your **daily standup** of this or through a PM.
+**Once the Pull Request is approved** by the trainers, merge it into master and create another branch from master to continue work.
+*Repeat ad infinitum* (until training end's)
 
 ## Environment
 
 You can work using your local environment:
 - You need to install [NodeJS](https://nodejs.org/en/) and [VSCode](https://code.visualstudio.com/download).
+- You can use [Webstorm 30 Day Trial](https://www.jetbrains.com/webstorm/) as an alternative to VSCode
```
Comment (antim-prisacaru) [inline README.md:44]: Don't mention the trial. Some people have university licenses or licenses paid by msg.

File: README.md [hunk 3]
```diff
@@ -51,25 +77,40 @@ The application will simply browse through a catalog of products. It will suppor
...
 
-Required Reading:
+#### Required:
+
+*Videos*
+
+1. [HTML in 100 Seconds](https://www.youtube.com/watch?v=ok-plXXHlWw)
+2. [CSS in 100 Seconds](https://www.youtube.com/watch?v=OEV8gMkCHXQ)
+3. [JavaScript in 100 Seconds](https://www.youtube.com/watch?v=DHjqpvDnNGE)
+4. [TypeScript in 100 Seconds](https://www.youtube.com/watch?v=zQnBQ4tB3ZA)
```
Comment (antim-prisacaru) [inline README.md:110]: Maybe add a why react sucks video as well to spite the react trainers. /s

File: README.md [hunk 4]
```diff
@@ -184,62 +257,89 @@ Online Shop:
 >
 > Add a new "Checkout" button on the shopping cart page, which creates a new order on the backend.
 
-Further Resources:
+#### Further Resources (Optional):
 
 - [CORS: Proxy-ing a backend server](https://angular.io/guide/build#proxying-to-a-backend-server)
 - [Http error handling](https://angular.io/guide/http#error-handling)
+- [Unsubscribe from RxJs Observables](https://www.thisdot.co/blog/best-practices-for-managing-rxjs-subscriptions)
```
Comment (antim-prisacaru) [inline README.md:264]: Might be worth suggesting somewhere to create an abstract class that can be inherited to deal with this.

File: README.md (hunk 2 above)
Comment (antonio-marasescu) [reply README.md:44]: And, they can use them if they have them?

File: README.md (hunk 3 above)
Comment (antonio-marasescu) [reply README.md:110]: :))) I hate things about both frameworks

File: README.md (hunk 4 above)
Comment (antonio-marasescu) [reply README.md:264]: Good point

Comment (antonio-marasescu) [reply README.md:264]: Done

File: README.md (hunk 1 above)
Comment (antonio-marasescu) [reply README.md]: removed it

File: README.md (hunk 2 above)
Comment (antonio-marasescu) [reply README.md]: Done

**Pull request merged**